x-pack/plugins/alerting/public/plugin.ts

```diff
@@ -75,21 +98,21 @@ export class AlertingPublicPlugin implements Plugin<PluginSetupContract, PluginS
 
   public start(core: CoreStart) {
     return {
-      getNavigation: async (alertId: Alert['id']) => {
-        const alert = await loadAlert({ http: core.http, alertId });
-        const alertType = await loadAlertType({ http: core.http, id: alert.alertTypeId });
+      getNavigation: async (ruleId: Alert['id']) => {
+        const rule = await loadRule({ http: core.http, ruleId });
+        const ruleType = this.ruleTypes!.find((type) => type.id === rule.alertTypeId);
```

Comment:
If `loadRuleTypes` failed to load, `this.ruleTypes` will be null so using `!` will cause a null pointer exception. Should we initialize `this.ruleTypes` to [] to avoid this?